### PR TITLE
G3 2024 v4 #14698

### PR DESCRIPTION
**Pull request by a18chrkr**
### Added PHP file with components
** Co-commit with @b22chran **

We have updated the stylesheet for the new installer and finalized a PHP-file with functions that represent complete components. We were however unsuccessful to link the php-file to the js-file "new_components.js". All file names and functions names are at the moment test-names and may be subjects to refactoring later on.

### CODE REVIEW

File: newinstaller/new_components.php [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            }
+        }
+
+        function breadCrumbDecr() {
+            stepSelected -= 1;
+            if(stepSelected<=1){
+                stepSelected = 1;
+            }
+        }
+    </script>
```
Comment (a22moamy) [inline newinstaller/new_components.php:156]: Please separate the javascript code from the HTML-code

**Review by a22moamy — CHANGES_REQUESTED** [review]
See the comment, then we can start testing this pull request :)

**Review by a22edvbu — APPROVED** [review]
All looks good. Great work!

File: newinstaller/style.css [hunk 2]
```diff
@@ -244,3 +253,32 @@ input[type=checkbox] {
...
+        height: .75rem;
+    }
+
+    .progressBarIndicator{
+        font-style: italic;
+        background-color: var(--secondary-color);
+        width: 15%;
+
+    }
+}
```
Comment (a22moamy) [inline newinstaller/style.css:284]: These selectors are nestled, is this SASS or is it a mistake? Rejecting it for now.

**Review by a22moamy — COMMENTED** [review]
See comment. If this is SASS and on purpose, please comment so and it will be approved.

File: newinstaller/style.css (hunk 2 above)
Comment (a22moamy) [reply newinstaller/style.css:284]: Seems like CSS and not SASS, please fix.

**Comment by d22mikwe** [comment]
NOT REAL MERGE. WAIT FOR OTHER MERGE

**Pull request merged**